lib/cli/commands/build.js

```diff
@@ -54,7 +54,7 @@ optimized output compatible with Node 6.
 
   build(dirs = this.dirs) {
     if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config' ];
+      dirs = this.dirs = [ 'app', 'config', 'blueprints' ];
```

Comment:
Maybe the solution would be to not transform anything inside `files` in a blueprint?